spec/routes/web/private_subnet_spec.rb

```diff
@@ -45,7 +45,7 @@
       it "can list no private subnets" do
         visit "#{project.path}/private-subnet"
 
-        expect(page.title).to eq("Ubicloud - Private Subnets")
+        expect(page.title).to eq("Ubicloud - Networking")
```

Comment:
Why don't we keep title as `Ubicloud - Private Subnets`? It is private subnets page  at all.